public/app/plugins/datasource/jaeger/webpack.config.ts

```diff
@@ -0,0 +1,17 @@
+import config from '@grafana/plugin-configs/webpack.config';
+
+const configWithFallback = async (env: Record<string, unknown>) => {
+  const response = await config(env);
+  if (response !== undefined && response.resolve !== undefined) {
+    response.resolve.fallback = {
+      ...response.resolve.fallback,
+      buffer: require.resolve('buffer/'),
+      events: require.resolve('events/'),
+      stream: require.resolve('stream-browserify'),
+      string_decoder: require.resolve('string_decoder/'),
```

Comment:
I don't believe this or the additional npm packages are necessary. See the [grafana webpack config](https://github.com/grafana/grafana/blob/main/scripts/webpack/webpack.common.js#L46-L53) where these are set to false.

Reply:
Yeah to be honest I don't understand why sometimes I have errors when building a package if I don't have these fallbacks while other times I don't. I guess I am either doing something wrong or I have an invalid state from time to time. Anyway, I'll check and remove them if not needed.

Reply:
It seems `stream` is needed 🤔